src/components/user/SignUpForm.js

```diff
@@ -54,8 +66,19 @@ class SignUpForm extends PureComponent {
             type="password"
           />
         </div>
-        <button type="submit">
-          <FormattedMessage id="login.form.submit" />
+        <div>
+          <Field
+            name="gender"
+            label={intl.formatMessage(messages.gender)}
+            component="select"
+            className="select"
+          >
+            <option>{intl.formatMessage(messages.selectGender)}</option>
+            {gender.map((option, key) => <option key={key} value={option.value}>{option.text}</option>)}
```

Comment:
Same here
```suggestion
            {gender.map(({ value, text }, key) => <option key={key} value={value}>{text}</option>)}
```

Reply:
I did't know you could do destructuring with a parameter like that, thanks!